Rocket.Chat/en.lproj/Localizable.strings

```diff
@@ -113,3 +113,8 @@
 
 // Members List
 "memberslist.title" = "Members (%d)";
+
+// Messages List
+"messageslist.title" = "Messages";
+"pinnedlist.title" = "Pinned Messages";
+"starredlist.title" = "Starred Messages";
```

Comment:
What do you think about using the same **key** style we're using in the other strings? This key would look like: 
- "memberslist.title" > "chat.members.list.title"
- "messageslist.title" > "chat.messages.list.title"
- "pinnedlist.title" > "chat.messages.pinned.list.title"
- "starredlist.title" >  "chat.messages.starred.list.title"